app/_locales/es/messages.json

```diff
@@ -1,10 +1,840 @@
 {
+  "accept": {
+    "message": "Aceptar"
+  },
+  "account": {
+    "message": "Cuenta"
+  },
+  "accountDetails": {
+    "message": "Detalles de la cuenta"
+  },
+  "accountName": {
+    "message": "Nombre de la cuenta"
+  },
+  "address": {
+    "message": "Direccion"
+  },
+  "addToken": {
+    "message": "Agregar Token"
+  },
+  "amount": {
+    "message": "Cantidad"
+  },
+  "amountPlusGas": {
+    "message": "Cantidad + Gas"
+  },
+  "appDescription": {
+    "message": "Extension del explorador usar Ethereum",
+    "description": "La descripcion de la aplicacion"
+  },
   "appName": {
     "message": "MetaMask",
-    "description": "The name of the application"
+    "description": "El nombre de la aplicacion"
   },
-  "appDescription": {
-    "message": "Administración de identidad en Ethereum",
-    "description": "The description of the application"
+  "attemptingConnect": {
+    "message": "Intentando conectar a la Blockchain"
+  },
+  "available": {
+    "message": "Disponible"
+  },
+  "back": {
+    "message": "Atras"
+  },
+  "balance": {
+    "message": "Balance:"
+  },
+  "balances": {
+    "message": "Tus balances"
+  },
+  "balanceIsInsufficientGas": {
+    "message": "Insuficiente balance para el gas total actual"
+  },
+  "beta": {
+    "message": "BETA"
+  },
+  "betweenMinAndMax": {
+    "message": "Debe ser mayor o igual a $1 y menor o igual a $2",
+    "description": "helper para ingresar hex como un ingreso decimal"
+  },
+  "borrowDharma": {
+    "message": "Pedir prestado con Dharma (Beta)"
+  },
+  "buy": {
+    "message": "Comprar"
+  },
+  "buyCoinbase": {
+    "message": "Comprar en Coinbase"
+  },
+  "buyCoinbaseExplainer": {
+    "message": "Coinbase es la forma mas popular en el mundo para comprar y vender bitcoin, ethereum, y litecoin."
```

Comment:
`Coinbase es la manera más popular en el mundo para comprar y vender bitcoin, ethereum y litecoin`. (forma -> manera, tilde and the last comma)